devtools/conda-envs/test_env_dgl_true.yaml

```diff
@@ -27,7 +27,7 @@ dependencies:
   - pyarrow
 
   # gcn
-  - dgl >=1.0
+  - dgl =2.1
```

Comment:
There's lot of stuff going on with GraphBolt in recent releases and similar stuff showed up in test failures: https://github.com/dmlc/dgl/releases

I got scared and basically guessed which works